Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -11,7 +11,17 @@ const { vtkWarningMacro, vtkErrorMacro, normalizeWheel, vtkOnceErrorMacro } =
 // Global methods
 // ----------------------------------------------------------------------------
 
-const EMPTY_MOUSE_EVENT = new MouseEvent('');
+let EMPTY_MOUSE_EVENT;
+if (typeof MouseEvent !== 'undefined') {
+  EMPTY_MOUSE_EVENT = new MouseEvent('');
+} else {
+  // Provide a fallback or handle differently for web worker context
+  EMPTY_MOUSE_EVENT = {
+    ctrlKey: false,
+    altKey: false,
+    shiftKey: false,
+  };
```

Comment:
I think we can just set EMPTY_MOUSE_EVENT to this object and do away with the `new MouseEvent` call.

Reply:
sure I can handle that